lib-extra/src/main/resources/com/diffplug/spotless/extra/eclipse_jdt_formatter/v4.11.0.lockfile

```diff
@@ -0,0 +1,19 @@
+# Spotless formatter based on JDT version 4.11.0 (see https://projects.eclipse.org/projects/eclipse.jdt)
+# Compare tag in M2 pom with https://git.eclipse.org/c/jdt/eclipse.jdt.core.git/log/?h=R4_9_maintenance to determine core version.
```

Comment:
It should state https://git.eclipse.org/c/jdt/eclipse.jdt.core.git/tag/?h=R4_11 now

Reply:
Done and updated to the four latest revisions